src/style.rs

```diff
@@ -348,7 +358,9 @@ impl ProgressStyle {
                         },
                     }
                 }
-                TemplatePart::Literal(s) => cur.push_str(s),
+                TemplatePart::Literal(s) => {
+                    cur.push_str(&s.replace('\t', &" ".repeat(self.tab_width)))
```

Comment:
How about we pre-replace tabs in literals after parsing? Would probably warrant a warning in the `with_tab_size()` docs that you should call it before `template()`, but would save on the work being done here in a somewhat hot path.

Reply:
Perhaps we could save the original template string and re-expand it when with_tab_width is called? That way user doesn't have to worry about the order of calls.

Reply:
That seems reasonable, yes.